### PR TITLE
Add support for Facebook Limited Login

### PR DESCRIPTION
**Pull request by miguel-s**
Facebook recently introduced a new login path called Limited Login. 

It prevents the user from being tracked by Facebook, but also does not grant access to Facebook's Graph API. The implementation is based on OpenID, similar to Apple's SignIn. You can find the docs here: [Docs](https://developers.facebook.com/docs/facebook-login/limited-login)

### How to use
Classic Graph API login (no change)
```Javascript
{
  id: "your_facebook_id",
  access_token: "facebook_graph_api_access_token"
}
```

Limited Login
```Javascript
{
  id: "your_facebook_id",
  token: "openid_token"
}
```

### Note
The implementation of OpenID auth adapters is very similar and we might want to DRY it up e.g facebook limited and apple signin.

### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [ ] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-server/issues?q=is%3Aissue).

### TODOs before merging

- [x] Add test cases
- [x] Add entry to changelog
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)

### CODE REVIEW

**Comment by dplewis** [comment]
Thanks for the PR. I didn't know about Limited Login. It looks good so far.

**Review by dplewis — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ ___
 - FIX: Fix error when a not yet inserted job is updated [#7196](https://github.com/parse-community/parse-server/pull/7196). Thanks to [Antonio Davi Macedo Coelho de Castro](https://github.com/davimacedo).
 - FIX: request.context for afterFind triggers. [#7078](https://github.com/parse-community/parse-server/pull/7078). Thanks to [dblythy](https://github.com/dblythy)
 - FIX: Winston Logger interpolating stdout to console [#7114](https://github.com/parse-community/parse-server/pull/7114). Thanks to [dplewis](https://github.com/dplewis)
+- NEW: Support Facebook Limited Login [#7219](https://github.com/parse-community/parse-server/pull/7219). Thanks to [miguel-s](https://github.com/miguel-s)
```
Comment (mtrezza) [inline CHANGELOG.md:33]: Can you please re-order this entry to the other "new"?

File: spec/AuthenticationAdapters.spec.js [hunk 2]
```diff
@@ -92,6 +92,7 @@ describe('AuthenticationProviders', function () {
         await provider.validateAuthData({ id: 'userId' }, params);
         params.appVersion = '5.123';
       }
+
```
Comment (mtrezza) [inline spec/AuthenticationAdapters.spec.js:95]: Remove empty line

**Review by mtrezza — CHANGES_REQUESTED** [review]
Just cosmetics

File: CHANGELOG.md (hunk 1 above)
Comment (miguel-s) [reply CHANGELOG.md]: Done

File: spec/AuthenticationAdapters.spec.js (hunk 2 above)
Comment (miguel-s) [reply spec/AuthenticationAdapters.spec.js]: Done

**Review by mtrezza — APPROVED** [review]
Thanks, LGTM!

**Comment by mtrezza** [comment]
Could you just merge the current master into this branch for the checks to pass?

**Comment by miguel-s** [comment]
@mtrezza Done :)

**Comment by mtrezza** [comment]
Thanks, I restarted the tests.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0-beta.1](https://github.com/parse-community/parse-server/releases/tag/5.0.0-beta.1)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [5.0.0](https://github.com/parse-community/parse-server/releases/tag/5.0.0)

**Pull request merged**